apps/atqr/api/src/app/infra/email/mailer.service.test.ts

```diff
@@ -42,10 +44,7 @@ describe('Mailer', () => {
 
   describe('Constructor', () => {
     it('Should build correctly', () => {
-      // Arrange
       const mailer = new Mailer(configService);
-      // Act
-      // Assert
       expect(mailer).toBeDefined();
```

Comment:
Being able to instatiate an object is a given, it is a feature of the language we are coding in so there's no need to test it. In some cases you may want to test if the object is instatiated correctly, meaning the business rules are being checked. In that case you need to check the properties of the object respect the rules and not that it simply exists